### DISCUSSION TITLE
OpenDeck v1 Board

### DISCUSSION THREAD

**Discussion by gh0stless** [shanteacontrols/OpenDeck#80] [Q&A]
I have a V.1 board (1.2.2) with firmware 4.0.6. 4.1.3 is the last version for this board? How do I flash them, apparently the bootloader is missing from me, the thing with Jumper BTLDR does not work for me. I have a TL866II + but it doesn't know the MEGA32U4.

**Comment by gh0stless** [comment]
And where can I find the settings for the fuces?

**Comment by paradajz** [comment]
4.1.3 is the last release for your board. To flash it, reboot the board into bootloader mode. I'm not sure what do you mean with "Jumper BTLDR does not work for me". I have also never heard of "TL866II".

**Reply by gh0stless** [reply]
thanks for the quick reply, i am a little confused, there is no firmeware section in my web configurator

**Comment by gh0stless** [comment]
I was able to find the values for the fuses. I flashed with a STK500 and Microchip Studio. A small info corner for the V1 board in the wiki would not be bad. thanks
